### PR TITLE
Adding embed guide

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
Staged: https://coda-packs-sdk--1801.com.readthedocs.build/en/1801/guides/advanced/embeds/

### CODE REVIEW

File: docs/guides/advanced/embeds.md
Comment (spencer-codaio) [inline docs/guides/advanced/embeds.md:52]: maybe note that this is a once per domain / per user disabled state (i.e. once you approve that domain you will not have to approve again in future, similarly if you approve it does not mean others will not see it)

**Review by spencer-codaio — APPROVED** [review]
looks great to me otherwise! thanks for this guide :)

File: docs/guides/advanced/embeds.md
Comment (ekoleda-codaio) [reply docs/guides/advanced/embeds.md:52]: Great callout! I meant to include that and forgot, and also didn't actually know the scope. Done.

**Pull request merged**
